pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md

```diff
@@ -22,100 +22,17 @@ This release provides new features and enhancements to improve the user experien
 
 If you have DC/OS deployed in a production environment, see [Known Issues and Limitations](#known-issues-and-limitations) to see if any potential operational changes for specific scenarios apply to your environment.
 
-- Updated DC/OS UI to master+v2.150.2.
```

Comment:
Why are you removing all of this if your commit message claims you only add something regarding domain sockets? This looks like an error to me.

Reply:
Sorry @bbannier maybe should have done a separate PR. This is our first release notes item for 2.1, so I needed to clean up the file for next release.

Reply:
Let's at least pull the cleanup into a separate commit with its own description. That would avoid any confusion whether the change was intentional or not.